src/database/models/QuoteModel.ts

```diff
@@ -16,4 +16,13 @@ export const Quote = new Schema({
 
 const quoteModel = model<QuoteI>('Quote', Quote, 'Quotes');
 
+export const addQuote = async (guildId: string, quote: string, author: string) => {
+  await quoteModel.create({
+    guildId: guildId,
+    quote: quote,
+    author: author,
+    date: Date.now(),
```

Comment:
This would mean that a quote is generated on model generation rather than when the quote is actually quoted, i.e. if someone goes back and approves a quote days old then the timestamp will be in correct